cpp/simulations/README.md

```diff
@@ -6,5 +6,9 @@ get into play once predefined incidence thresholds (50 and 200) are exceeded.
 
 - 2021_vaccination_delta: Extending the model of 2020_npis_wildtype by vaccination and reinfection and
 considering the effect of vaccination in combination with the lifting of NPIs during the arrival of Delta.
+The data which is necessary to run these simulations is generated by the MEmilio epidata package.
+A simple generation is possible with the script `data_generation.sh`.
+There only the path to the Python environment must be given. The path can either be set in the script or we call
+the script directly with the option "-PATH_ENV PATH/TO/VIRTUAL/ENV".
```

Comment:
```suggestion
The data which is necessary to run these simulations is generated by the MEmilio epidata package.
This can be done by calling the particular python routines manually or by running the `data_generation.sh` shell script.
The shell script assumes the existence of a python environment and only the path activation function of the environment 
must be given to the script. The path can either be set in the script or by calling the script directly with the option
"-PATH_ENV PATH_TO_VIRTUAL_ENV/ACTIVATE/ACTIVATION_EXEC".
The script needs to be executed from the `simulations` folder directly by `./data_generation.sh`.
```